agent/src/iconfucius/siwb.py

```
@@ -58,6 +58,94 @@
+            pem_content = f.read()
+        wallet_identity = Identity.from_pem(pem_content)
+        principal_text = str(wallet_identity.sender())
+
+        client = Client(url=IC_HOST)
+        anon_agent = Agent(Identity(anonymous=True), client)
+        from iconfucius.transfers import create_icrc1_canister, get_balance
+        ckbtc = create_icrc1_canister(anon_agent)
+        balance = get_balance(ckbtc, principal_text)  # in e8s
+        balance_sats = balance  # ckBTC balance is in e8s = sats
+        result = balance_sats >= min_sats
+    except Exception:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)